docs/en/05_Contributing/05_Making_A_SilverStripe_Core_Release.md

```diff
@@ -337,6 +339,8 @@ including:
 * Security fixes included
 * Major changes
 
+Before publication, ensure that the release plan has been peer reviewed by another member of the core team.
```

Comment:
By the way, I added this because it's part of our CWP process and I think it's important to have. If you have lock step then everything's the same so you don't need a sanity check, but with different module versions you do need someone to check that everything (A) needs to be released and (B) is the right version to release